packages/ts/hilla-file-router/package.json

```diff
@@ -47,27 +50,32 @@
   "files": [
     "*.{d.ts.map,d.ts,js.map,js}",
     "runtime",
+    "shared",
     "vite-plugin"
   ],
   "publishConfig": {
     "access": "public"
   },
   "peerDependencies": {
-    "react-router": "^6.21.1"
+    "react": "^18.2.0",
+    "react-router": "^6.21.1",
+    "react-router-dom": "^6.21.1"
   },
   "devDependencies": {
     "@esm-bundle/chai": "^4.3.4-fix.0",
+    "@types/chai-as-promised": "^7.1.8",
     "@types/chai-like": "^1.1.3",
     "@types/deep-equal-in-any-order": "^1.0.3",
     "@types/mocha": "^10.0.6",
-    "@types/sinon": "^17.0.3",
+    "@types/sinon": "^10.0.17",
+    "chai-as-promised": "^7.1.1",
     "chai-like": "^1.1.1",
     "deep-equal-in-any-order": "^2.0.6",
     "mocha": "^10.2.0",
     "rimraf": "^5.0.5",
-    "sinon": "^17.0.1",
+    "sinon": "^16.0.0",
     "type-fest": "^4.9.0",
-    "typescript": "^5.3.3"
+    "typescript": "5.3.2"
```

Comment:
nit: since the plugin uses TS API, `typescript` is a runtime dependency

Reply:
Right, thanks